docs/mobile.md

```diff
@@ -0,0 +1,115 @@
+---
+id: mobile
+title: "LDK for Lightning mobile devices"
+---
+
+## Introduction
+
+This documents covers the challenges of building a mobile-first Lightning node and what
+solution LDK offers. Note, this documentation already supposes a reader familiarity with
+the Lightning protocol and Bitcoin fundamental notions.
+
+* [Fee Estimation](#fee-estimation) covers issues related to fee-estimation
+* [Liveliness & Watchtowers](#watchtowers) covers issues related to Lightning liveliness requirement
+* [Chain-validation](#chain-validation) covers issues related to chain validation
+
+## Fee Estimation
+
+A Lightning node can broadcast four types of transactions. A _funding transaction_, to open one or
+multiple channel transactions. A _penalty transaction_ in reaction of a cheating by a counterparties
+to double-spend a channel. A _commitment transaction_, either motivated by reallocating funds
+locked in this channel to another purpose or in reaction to a in-flight HTLC to claim onchain. Such
+unilateral broadcast should only happen when the counterparty is faulting to cooperate. A _closing 
+transaction_, cooperatively built by parties to complete the channel.
+
+Block space being a scarce resources, a Lightning node, similar to any other kind of Bitcoin applications
+is in competition to confirm its transactions. While bidding to get its transaction included, a
+Bitcoin application's fee-estimation should be as accurate as possible to save in fees. Further, a 
+Lightning node, has a time-sensitive requirement with regards to confirmation. Commitment and penalty 
+transactions to produce their effect, must be confirmed before a block height timelock expire.
+Beyond timelock expiration the counterparty can confirm transactions at its advantage, thus
+provoking a loss of funds of the Lightning user.
+
+Contrary to simple Bitcoin transactions, of which the delaying of confirmation isn't directly
+provoking losses, in Lightning fee-estimation accuracy is a safety-critical requirement.
+
+A Bitcoin full-node implements its fee-estimation on top of its mempool, by observing deltas
```

Comment:
This is a little hard to read. Maybe "Bitcoin Core implements fee-estimation in a way which attempts to be robust against various censorship attacks by analyzing transactions that were both in its mempool and were later confirmed. By avoiding waiting for confirmation, other fee estimators may respond faster to fee spikes. Due to its immediate confirmation pressure, Lightning may benefit from a fee estimator more in the second category, but care must be taken to avoid P2P censorship attacks. LDK attempts to deal with insufficient fee estimation by using RBF and slowly increasing transaction fees until confirmation."